account_bank_receipt/models/account_bank_receipt.py

```diff
@@ -0,0 +1,378 @@
+            if deposit.company_id.currency_id != deposit.currency_id:
+                currency_none_same_company_id = deposit.currency_id.id
+            for line in deposit.receipt_line:
+                count += 1
+                total += line.allocation
+            if deposit.move_id:
+                for line in deposit.move_id.line_ids:
+                    if line.debit > 0 and line.reconciled:
+                        reconcile = True
+            deposit.total_amount = total
+            deposit.is_reconcile = reconcile
+            deposit.currency_none_same_company_id =\
+                currency_none_same_company_id
+            deposit.check_count = count
+
+    name = fields.Char(
+        size=64,
+        default='/',
+        readonly=True,
+        copy=False,
+    )
+    receipt_date = fields.Date(
+        required=True,
+        states={'done': [('readonly', '=', True)]},
+        default=fields.Date.context_today,
+    )
+    journal_id = fields.Many2one(
+        comodel_name='account.journal',
+        domain=[('type', '=', 'bank'), ('bank_account_id', '=', False)],
+        required=True,
+        states={'done': [('readonly', '=', True)]},
+        copy=False,
+    )
+    journal_default_account_id = fields.Many2one(
+        comodel_name='account.account',
+        string='Default Debit Account of the Journal',
+        related='journal_id.default_debit_account_id',
+    )
+    currency_id = fields.Many2one(
+        comodel_name='res.currency',
+        required=True,
+        states={'done': [('readonly', '=', True)]},
+    )
+    currency_none_same_company_id = fields.Many2one(
+        comodel_name='res.currency',
+        string='Currency (False if same as company)',
+        compute='_compute_bank_receipt',
+        store=True,
+    )
+    state = fields.Selection([
+        ('draft', 'Draft'),
+        ('done', 'Done')],
+        string='Status',
+        default='draft',
+        readonly=True,
+    )
+    move_id = fields.Many2one(
+        comodel_name='account.move',
+        string='Journal Entry',
+        readonly=True,
+    )
+    bank_journal_id = fields.Many2one(
+        comodel_name='account.journal',
+        string='Bank Account',
+        required=True,
+        domain="[('company_id', '=', company_id), ('type', '=', 'bank'), "
+               "('bank_account_id', '!=', False)]",
+        states={'done': [('readonly', '=', True)]}
+    )
+    line_ids = fields.One2many(
+        comodel_name='account.move.line',
+        related='move_id.line_ids',
+        string='Lines',
+        readonly=True,
+    )
+    company_id = fields.Many2one(
+        comodel_name='res.company',
+        required=True,
+        states={'done': [('readonly', '=', True)]},
+        default=lambda self: self.env['res.company']._company_default_get(),
+    )
+    total_amount = fields.Float(
+        compute='_compute_bank_receipt',
+        readonly=True,
+        store=True,
+        digits=dp.get_precision('Account'),
```

Comment:
No 'Account', please use 'Product Price'
Note: you can check availability in decimal_precision table.